phyui/plot/waveforms.py

```diff
@@ -325,6 +326,7 @@ class WaveformView(PanZoomCanvas):
     def __init__(self, **kwargs):
         super(WaveformView, self).__init__(**kwargs)
         self.visual = Waveforms()
+        self.resizable = Bool(defaultValue=True, sync=True)
```

Comment:
Can you send me the link where you found defaultValue? Maybe it was in JS? I've never seen this camel case convention in a mainstream Python project.

So I think the widget is currently in VisPy, which is probably not a great option. What we want is some base class in VisPy deriving from DOMWidget, that we can specialize elsewhere like in phyui.